addons/sale_coupon_delivery/__init__.py

```diff
@@ -1,12 +1,3 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, SUPERUSER_ID
-
 from . import models
-
-
-def uninstall_hook(cr, registry):
-    env = api.Environment(cr, SUPERUSER_ID, {})
-    env['delivery.carrier'].search([
-        ('delivery_type', '=', 'temando')
-    ]).write({'delivery_type': 'fixed', 'fixed_price': 0})
```

Comment:
todo: adapt selection field extension

Reply:
and remove from manifest

Reply:
This was a leftover, this `temando` delivery no longer exists, or at least I did not find any references to it (other than within po(t) files)